UI/window-basic-status-bar.cpp

```diff
@@ -233,11 +223,34 @@ void OBSBasicStatusBar::UpdateCPUUsage()
 
 	QString text;
 	text += QString("CPU: ") +
-		QString::number(main->GetCPUUsage(), 'f', 1) + QString("%, ") +
-		QString::number(obs_get_active_fps(), 'f', 2) + QString(" fps");
+		QString::number(main->GetCPUUsage(), 'f', 1) + QString("%");
+
+	statusWidget->ui->cpuUsage->setText(text);
+	statusWidget->ui->cpuUsage->setMinimumWidth(
+		statusWidget->ui->cpuUsage->width());
 
-	cpuUsage->setText(text);
-	cpuUsage->setMinimumWidth(cpuUsage->width());
+	UpdateCurrentFPS();
+}
+
+void OBSBasicStatusBar::UpdateCurrentFPS()
+{
+	OBSBasic *main = qobject_cast<OBSBasic *>(parent());
+	if (!main)
+		return;
```

Comment:
This can't ever be `NULL` I don't think.
Either way, the variable is unused here and can get removed.